config/extension_import_variables.go

```diff
@@ -76,7 +76,7 @@ func (item *ImportVariables) loadVariables(currentVariables map[string]interface
 		}
 		item.options().ImportVariablesMap(imported, source)
 		level := *item.FlattenLevels
-		if level > 0 && len(nested) > 1 && nested == "" {
+		if level > 0 && len(item.NestedObjects) > 1 && nested == "" {
```

Comment:
Here, I made a big mistake, I checked the length of the string instead of the length of the list. The desired effect is to remove one level to the flattened levels when we specify a list of prefix AND an empty prefix. The empty one should reduce the flattening level by one in that specific case. By not doing it, we generated variables such as:
`project_subproject_variable_coveo:environment` while we should have stopped at `project_subproject_variable`.

The condition was in fact impossible to reach. I did not add unit test to catch it in the future since Julien removed all this flattening stuff in the master branch.